_posts/technotes/2017-09-07-spelling-release.md

```diff
@@ -0,0 +1,88 @@
+---
+name: spelling-release
+layout: post_discourse
+title: "Spelling 1.0: quick and effective spell checking in R"
+date: 2017-09-07
+authors:
+  - name: Jeroen Ooms
+categories: blog
+tags:
+- r
+- packages
+- software
+- spelling
+- text
+---
+
+
+The new rOpenSci [spelling](https://cran.r-project.org/web/packages/spelling/index.html) package provides utilities for spell checking common document formats including latex, markdown, manual pages, and description files. It also includes tools especially for package authors to automate spell checking of R documentation and vignettes.
+
+## Spell Checking Packages
+
+The main purpose of this package is to quickly find spelling errors in R packages. Use `spell_check_package()` function extracts all text from your package manual pages and vignettes, compares it against a language (e.g. en_US or en_GB), and lists potential errors in a nice tidy format:
```

Comment:
"Use `spell_check_package()` function ..." doesn't sound right, did you mean "The `spell_check_package()` function ..."?

Reply:
Good catch, that's fixed now!